exporters/common/src/main/java/io/opentelemetry/exporter/internal/ExporterBuilderBasics.java

```diff
@@ -0,0 +1,29 @@
+package io.opentelemetry.exporter.internal;
+
+import io.opentelemetry.sdk.common.export.MemoryMode;
+import io.opentelemetry.sdk.common.export.RetryPolicy;
+import javax.annotation.Nullable;
+import java.time.Duration;
+
+/**
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ *  * at any time.
+ * @param <T> The specific concrete builder type
+ */
+public interface ExporterBuilderBasics<T> {
```

Comment:
new interface intended to represent the most common shared methods across builders.

Reply:
I like that idea :smile: